OpenOversight/app/db_repository/versions/006_migration.py

```diff
@@ -0,0 +1,42 @@
+from sqlalchemy import *
+from migrate import *
+
+
+from migrate.changeset import schema
+pre_meta = MetaData()
+post_meta = MetaData()
+unit_types = Table('unit_types', post_meta,
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('descrip', String(length=120)),
+    Column('department_id', Integer),
+)
+
+officers = Table('officers', post_meta,
+    Column('id', Integer, primary_key=True, nullable=False),
+    Column('last_name', String(length=120)),
+    Column('first_name', String(length=120)),
+    Column('middle_initial', String(length=120)),
+    Column('race', String(length=120)),
+    Column('gender', String(length=120)),
+    Column('employment_date', DateTime),
```

Comment:
this has the potential for weird timezone bugs, but i don’t know how much potential it has.

Specifically i don’t remember if postgres’s datetime support includes zone or assumes the application handles it; there’s a potential for discrepancy when initializing these with dates (at time 00:00) and being returned referencing the day before/after we intended.